src/lib/github/getCheckedComments.ts

```diff
@@ -0,0 +1,16 @@
+import { getBasicRepoProps } from "./utils";
+
+import { Context } from "./types";
+
+export default async function getCheckedComments (context: Context): Promise<string[]> {
```

Comment:
Can you process the returned strings such that you have an `array` of this type below;
```
 interface GhIssueComment {  
    file: string,
    comments: string[]
 }
```

Reply:
is the file name necessary since in the comment string includes the file name.

Reply:
split it up the comment string such that the file name is assigned to file and comment text to comment.